src/confirm.rs

```diff
@@ -21,10 +25,21 @@ pub struct Confirm {
 
 impl Confirm {
     pub fn run(&self) -> Result<()> {
-        let value = dialoguer::Confirm::with_theme(&ColorfulTheme::default())
-            .with_prompt(&self.message)
-            .default(self.default)
-            .interact()?;
+        let theme = ColorfulTheme::default();
+        let mut input = dialoguer::Confirm::with_theme(&theme);
+
+        input.with_prompt(&self.message).default(self.default);
+
+        let ret = if self.cancel {
+            input.interact_opt()?
+        } else {
+            Some(input.interact()?)
+        };
+
+        let value = match ret {
+            Some(value) => value,
+            None => self.default,
```

Comment:
process::exit here to be consistent with other prompts below. No need to add `default` option.

Reply:
Oh right, Enter without `y|n` will be `self.default`.